### PR TITLE
Responsive version branch created

### PR DESCRIPTION
**Pull request by ZahraArshia**
HELLO 😄 
responsive version of portfolio added.

### CODE REVIEW

File: zahra-portfolio.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                <ul class="tag">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="action">
+
+                    <button class="button" type="button">
+                            See Project
+                        </button>
```
Comment (ShoiraTa) [inline zahra-portfolio.html:70]: - [ ] Please update button styles on hover 
> Did you know that you are provided with button styles on figma? 

image

File: zahra-portfolio.html [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                    </ul>
+                </div>
+
+            
+        </div>
+       
+        </section>
+        <footer>
+          <section class="wrapper">
+            <h2 class="contact-title">Contact me</h2>
```
Comment (ShoiraTa) [inline zahra-portfolio.html:224]: - [ ] Contact me section has a  white line background visible on both sides and footer part:
image

- [ ]  please kindly make the `purple` color contact me section spread all along the contact section like it was given in figma 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ZahraArshia ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZahraArshia** [comment]
Hi,
- main section paddings removed, so there is no space between body and contact page background now.
- button styles updated based on figma template.

**Comment by ZahraArshia** [comment]
Hi,
- main section paddings removed, so there is no space between body and contact page background now.
- button styles updated based on figma template.

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi again @ZahraArshia ,  

Your project is complete! 💯🌟 
All looks great now, thank you for working on requested changes 😉 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**